### ISSUE TITLE
Color schemes do not match their previews

### ISSUE THREAD

**Issue by mikeymop** [Microsoft/console#389]
This bug-tracker is monitored by Windows Console development team and other technical types.  **We like detail!**

If you have a feature request, please post to [the UserVoice](https://wpdev.uservoice.com/forums/266908).  

Please use this form and describe your issue, concisely but precisely, with as much detail as possible

* Your Windows build number:  10.0.17134.648]

* What you're doing and what's happening: (Copy & paste specific commands and their output, or include screen shots)
.\ColorTool campbell
.\ColorTool solarized_dark

* What's wrong / what should be happening instead:
When previewing themes with `ColorTool -s` I see a lot of attractive dark themes such as solarized dark and deuteranopia.

I proceed to enable these colors with and without the -x option.
However the color schemes that are actually set as a result of this look nothing like the previews.

For example, solarized dark gives me a pink background with dark pink text and purple folder names.
Deuteranopia gives me a dark color with highlight and normal text very close in color making it impossible to read from the terminal.

I want to use color tool to get rid of the terrible default color scheme that uses dark blue on black however the results are not what appears on the post that introduced me to colortool.

**Comment by zadjii-msft** [comment]
See [this thread](https://github.com/Microsoft/console/issues/23#issuecomment-325179258) for an explanation.

tl;dr: Powershell has "Magenta" set as the default bg color, you need to go into the properties and change that manually.
